### PR TITLE
Use color library across the platform

### PR DESCRIPTION
**Pull request by LuizFNJ**
# Description
*Some colors on the platform were being defined directly within the components. I’m fixing this by replacing those definitions with the colors specified in the colors component.*

Fixes # (issue)

# Type of change
*Please delete options that are not relevant.*

- [ ] Bug fix (non-breaking change which fixes an issue)     
- [ ] New feature (non-breaking change which adds functionality)     
- [x] Existing feature enhancement (non-breaking change which modifies existing functionality)     

# Testing
*I believe it's just a matter of checking if the colors are correct in the components that were adjusted.*

# Developer Checklist
## General
- [ ] Code is appropriately commented, particularly in hard-to-understand areas     
- [ ] Repository documentation has been updated (Readme.md) with additional steps required for a local environment setup.
- [x] No `console.log` or related logging is added.
- [x] No code is repeated/duplicated in violation of DRY. The exception to this is for new (MVP/Prototype) functionality where the abstraction layer may not be clear (comments should be added to explain the violation of DRY in these scenarios).   
- [ ] Documented with TSDoc all library and controller new functions

## Frontend Changes
- [ ] No new styling is added through CSS files (Unless it's a bugfix/hotfix)
- [x] All types are added correctly

## Backend Changes
- [ ] All endpoints are appropriately secured with Middleware authentication
- [ ] All new endpoints have a interface schema defined

### Tests
- [ ] All existing unit and end to end tests pass across all services     
- [ ] Unit and end to end tests have been added to ensure backend APIs behave as expected     

# Merge Request Review Checklist 
- [ ] An issue is linked to this PR and these changes meet the requirements outlined in the linked issue(s)     
- [ ] High risk and core workflows have been tested and verified in a local environment.     
- [ ] Enhancements or opportunities to improve performance, stability, security or code readability have been noted and documented in JIRA issues if not being addressed.     
- [ ] Any dependent changes have been merged and published in downstream modules     
- [ ] Changes to multiple services can be deployed in parallel and independently. If not, changes should be broken out into separate merge requests and deployed in order.

closes#1393

### CODE REVIEW

File: src/components/Collaborative/Comment/CommentCard.style.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const CommentCardStyle = styled.div`
         props.isselected === "true" ? `translateX(-15px)` : `translateX(0)`};
 
     &:hover {
-        box-shadow: 0px 3px 3px 1px rgba(0, 0, 0, 0.2);
+        box-shadow: 0px 3px 3px 1px ${colors.lightGraySecondary};
```
Comment (pepermao) [inline src/components/Collaborative/Comment/CommentCard.style.tsx:22]: suggestion: use the same standard color as above `colorShadow`

File: src/components/Form/InputSearch.tsx [hunk 2]
```diff
@@ -9,16 +9,16 @@ const InputSearchStyled = styled(Input.Search)`
...
     span.ant-input-affix-wrapper {
         background: ${({ backgroundColor = colors.lightGray }) =>
-            backgroundColor};
-        box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
+        backgroundColor};
+        box-shadow: 0px 2px 2px ${colors.colorShadow};
         border-radius: 4px;
         &:focus-within {
-            border-color: #d9d9d9;
+            border-color: ${colors.lightBluePrimary};
```
Comment (pepermao) [inline src/components/Form/InputSearch.tsx:16]: issue: Those color aren't the same

File: src/components/Modal/AletheiaModal.style.tsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ const AletheiaModal = styled(Modal)`
     }
 
     .hide-modal {
-        color: #db9f0d;
+        color: ${colors.warning};
```
Comment (pepermao) [inline src/components/Modal/AletheiaModal.style.tsx:60]: suggestion: We have two identical color with different names, we should pick only one to use across the platform

`lightYellow` and `warning`

File: src/styles/colors.ts [hunk 4]
```diff
@@ -12,10 +12,11 @@ const colors = {
     lightBlueMain: "rgb(79, 141, 180)", //#4F8DB4
     lightBlueSecondary: "rgb(103, 190, 242)", //#67bef2
     white: "rgb(255, 255, 255)", //#ffffff
+    colorShadow: "rgba(0, 0, 0, 0.25)", //#00000040
     blackPrimary: "rgb(17, 17, 17)", //#111111
     blackSecondary: "rgb(81, 81, 81)", //#515151
     blackTertiary: "rgb(32, 34, 34)", // #202222
-    lightYellow: "rgba(219, 159, 13, 0.3)", //#db9f0d
+    lightYellow: "rgba(219, 159, 13, 0.3)", //#db9f0d4d
```
Comment (pepermao) [inline src/styles/colors.ts:19]: issue: This color does not exist

**Review by pepermao — CHANGES_REQUESTED** [review]

File: src/components/Form/InputSearch.tsx (hunk 2 above)
Comment (LuizFNJ) [reply src/components/Form/InputSearch.tsx:16]: using "border-color: ${colors.grayTertiary}" 

before 
Captura de tela de 2024-10-14 21-42-43
after
Captura de tela de 2024-10-14 21-43-19

**Review by thesocialdev — APPROVED** [review]

**Review by pepermao — APPROVED** [review]

**Pull request merged**